app/jobs/author_harvest_job.rb

```diff
@@ -1,24 +1,52 @@
 class AuthorHarvestJob < ActiveJob::Base
   queue_as :default
 
-  ##
   # Performs an asynchronous harvest and save for a given Author
   # @param [String] cap_profile_id
   # @param [Boolean] harvest_alternate_names
+  # @return [void]
   def perform(cap_profile_id, harvest_alternate_names: false)
-    harvester = ScienceWireHarvester.new
-    harvester.use_author_identities = harvest_alternate_names
-    author = Author.where(cap_profile_id: cap_profile_id).first
+    author = Author.find_by(cap_profile_id: cap_profile_id)
     author ||= Author.fetch_from_cap_and_create(cap_profile_id)
-    harvester.harvest_pubs_for_author_ids author.id
-    pubs = Contribution.where(author_id: author.id).map(&:publication).each do |p|
-      logger.info "publication #{p.id}: #{p.pub_hash[:apa_citation]}"
-    end
-    logger.info "Number of publications #{pubs.count}"
+    raise "Could not find or fetch author: #{cap_profile_id}" unless author.is_a?(Author)
+    sciencewire(author, harvest_alternate_names)
+    web_of_science(author, harvest_alternate_names)
   rescue => e
     msg = "AuthorHarvestJob.perform(cap_profile_id=#{cap_profile_id}, harvest_alternate_names=#{harvest_alternate_names})"
```

Comment:
I know you didn't touch this line, but I think the log message intends to have `harvest_alternate_names:`, not `harvest_alternate_names=`...

Reply:
Yea, it's a bit weird for a log message to appear to construct the call, as if you could cop-n-pasta to run it again.  If that's the intention, it _could_ be fixed up.